hbase/src/main/java/com/yahoo/ycsb/db/HBaseClient.java

```diff
@@ -316,11 +316,12 @@ public int update(String table, String key, HashMap<String,ByteIterator> values)
         Put p = new Put(Bytes.toBytes(key));
         for (Map.Entry<String, ByteIterator> entry : values.entrySet())
         {
+            byte[] value = entry.getValue().toArray();
             if (_debug) {
                 System.out.println("Adding field/value " + entry.getKey() + "/"+
-                  entry.getValue() + " to put request");
+                  Bytes.toString(value) + " to put request");
```

Comment:
please use Bytes.toHexString, since the workload might not use a value that can be turned into a string.